projects/dsp-ui/src/lib/action/components/login-form/login-form.component.html

```diff
@@ -0,0 +1,69 @@
+<div class="login-container" *ngIf="!session">
+
+    <h2 class="login-form-title mat-title">{{formLabel.title}}</h2>
+
+    <!-- This is the login form -->
+    <form [formGroup]="form" class="login-form">
+
+        <!-- Username -->
+        <mat-form-field class="full-width login-field">
+            <!-- <mat-icon matPrefix>person</mat-icon> -->
```

Comment:
If we are not using the icon, you should remove the line. Same in line 24.

Reply:
This is the question: Should we display nice icons in the form field or not?

Reply:
![Screenshot 2020-07-17 at 06 09 09](https://user-images.githubusercontent.com/4253580/87747613-1d8dc780-c7f4-11ea-81fc-2928581e0a18.png)

VS

![Screenshot 2020-07-17 at 06 09 19](https://user-images.githubusercontent.com/4253580/87747616-1ebef480-c7f4-11ea-8ac8-9ce749460c4a.png)

Reply:
I prefer the version without icons. So yes, I agree. @mdelez you can delete the commented mat-icons.

Reply:
I like the icons. Is this something we can set up so the user can choose to display it or not?

Reply:
> Is this something we can set up so the user can choose to display it or not?

Why not? Can you add an `@Input` "icons" (or something similar) of type boolean (default `false`) to show them or not...

Reply:
added in 32e144c